app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCTableView.scala

```diff
@@ -90,10 +96,41 @@ class DLCTableView(model: DLCPaneModel) {
       }
     }
 
+    val pnlCol = new TableColumn[DLCStatus, String] {
```

Comment:
these new columns make it so the table is larger than the default screen, maybe we should drop the my and counter party col columns in place

Reply:
Here is what i got it to on 872a9fe 

![Screenshot from 2021-05-31 08-57-16](https://user-images.githubusercontent.com/3514957/120204309-480b2800-c1ee-11eb-96c2-de885c565398.png)

Reply:
for me total col is a bit cut off and RoR is cut off a bit on both, imo it'd be better to drop the 2 collateral columns and give the others some room, we don't want it too cluttered

![image](https://user-images.githubusercontent.com/15256660/120205637-cd430c80-c1ef-11eb-9485-572fb762f013.png)